lib/exe/js.php

```diff
@@ -115,6 +115,7 @@ function js_out(){
     if(!empty($templatestrings)) {
         $lang['js']['template'] = $templatestrings;
     }
+    $lang['js']['media_update'] = $lang['media_update'];
```

Comment:
I understand that this is to make sure we can reuse `media_update`. However, I am not sure if we want this hack in the future.